src/styles/_stopwatch.scss

```diff
@@ -0,0 +1,71 @@
+main {
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50vmin;
+  position: relative;
+
+  &__minutes {
+    height: $minute-arrow-height;
+    width: $minute-arrow-width;
+    background-color: $minute-arrow-color;
+
+    @include position-absolute-center;
+
+    z-index: 1;
+
+    transform-origin: bottom;
+    animation: watch-start 3600s steps(60) infinite;
+  }
+
+  &__seconds {
+    height: $seconds-arrow-height;
+    width: $seconds-arrow-width;
+    background-color: $seconds-arrow-color;
+
+    @include position-absolute-center;
+
+    z-index: 2;
+
+    transform-origin: bottom;
+    animation: watch-start 10s linear infinite;
```

Comment:
Here should be 60s. You can also make variables with seconds and then overwrite the values inside selector with modifier.